src/org/typo3/chefci/v2/stages/Publish.groovy

```diff
@@ -21,6 +21,8 @@ class Publish extends AbstractStage {
      */
     protected publish() {
 
+        (new Slack(script)).notifyVersionBump()
```

Comment:
Something like this makes a lot of sense to me. Call that like a `NotificationService` thing or so?